packages/inflection-tables/src/paradigm/data/greek/greek-paradigm-dataset.js

```diff
@@ -24,6 +24,10 @@ export default class GreekParadigmDataset extends LanguageDataset {
       .map('future_perfect', [Constants.TENSE_FUTURE_PERFECT])
   }
 
+  static get language () {
+    return Language.GREEK
+  }
+
   static get languageID () {
```

Comment:
do we need languageID getter now?

Reply:
It's used in couple of places in the constructor (https://github.com/alpheios-project/alpheios-core/blob/dev-annotations-next/packages/inflection-tables/src/paradigm/data/greek/greek-paradigm-dataset.js#L15-L16), but we could possibly be replaced with a constant. The problem with `languageID` is that it's used all over the place in the inflection tables and it's hard to track those usages exactly. I've tried to reduce changes to inflection tables to the minimal amount as changes of this library are out of scope for the current annotation work. So maybe we'll leave it as is for now? We can later add a log statement to track its usage.